src/api/spec/models/obs_factory/staging_project_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe ObsFactory::StagingProject do
+  let(:distribution) { ObsFactory::Distribution.new(create(:project, name: 'openSUSE:Factory')) }
```

Comment:
:thought_balloon:  Why no factory?

Reply:
I already tried that before and there are some obstacles. I think this became easier with the change of the initializer. So I was planing to do that after this PR.